src/deepsparse/transformers/helpers.py

```diff
@@ -89,26 +89,25 @@ def get_onnx_path_and_configs(
             framework_dir = os.path.join(model_path, "pytorch")
         if framework_dir and os.path.isdir(framework_dir):
             framework_files = os.listdir(framework_dir)
-            if _MODEL_DIR_CONFIG_NAME in framework_files:
+            if MODEL_DIR_CONFIG_NAME in framework_files:
                 config_path = framework_dir
-            if _MODEL_DIR_TOKENIZER_NAME in framework_files:
+            if MODEL_DIR_TOKENIZER_NAME in framework_files:
                 tokenizer_path = framework_dir
 
         # prefer config and tokenizer files in same directory as model.onnx
-        if _MODEL_DIR_CONFIG_NAME in model_files:
+        if MODEL_DIR_CONFIG_NAME in model_files:
             config_path = model_path
-        if _MODEL_DIR_TOKENIZER_NAME in model_files:
+        if MODEL_DIR_TOKENIZER_NAME in model_files:
             tokenizer_path = model_path
 
     elif model_path.startswith("zoo:"):
         zoo_model = Model(model_path)
-        onnx_path = zoo_model.onnx_model.path
-        config_path = _get_file_parent(
-            zoo_model.deployment.default.get_file(_MODEL_DIR_CONFIG_NAME).path
-        )
-        tokenizer_path = _get_file_parent(
-            zoo_model.deployment.default.get_file(_MODEL_DIR_TOKENIZER_NAME).path
-        )
+
+        deployment = zoo_model.deployment.default
+        onnx_path = deployment.get_file(MODEL_DIR_ONNX_NAME).path
+        config_path = deployment.get_file(MODEL_DIR_CONFIG_NAME).path
+        tokenizer_path = deployment.default.get_file(MODEL_DIR_TOKENIZER_NAME).path
```

Comment:
was this tested? afaik we want the parent dir of the tokenizer/config as this is what's required to load the actual objects